xml/ha_hawk2_rsc_i.xml

```diff
@@ -0,0 +1,755 @@
+      <mediaobject>
+       <imageobject role="fo">
+        <imagedata fileref="hawk2-primitive-stonith.png" width="100%" format="PNG"/>
+       </imageobject>
+       <imageobject role="html">
+        <imagedata fileref="hawk2-primitive-stonith.png" width="100%" format="PNG"/>
+       </imageobject>
+      </mediaobject>
+     </figure>
+    </step>
+    <step>
+     <para>
+      Confirm your changes to create the &stonith; resource.
+     </para>
+     <para>A message at the top of the screen shows if the
+      action has been successful.</para>
+    </step>
+   </procedure>
+   <para>
+    To complete your fencing configuration, add constraints. 
+    For more details, refer to <xref linkend="cha.ha.fencing"/>.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec.conf.hawk2.rsc.group">
+   <title>Adding Cluster Resource Groups</title>
+   <para>
+    Some cluster resources depend on other components or resources and
+    require that each component or resource starts in a specific order and
+    runs on the same server. To simplify this configuration we support the
+    concept of groups.
+   </para>
+   
+   
+   <para>Resource groups contain a set of resources that need to be located together, be started
+    sequentially and stopped in the reverse order. For an example of a resource group and more
+    information about groups and their properties, refer to <xref
+    linkend="sec.ha.config.basics.resources.advanced.groups"/>.
+   </para>
+   <note>
+    <title>Empty Groups</title>
+    <para>
+     Groups must contain at least one resource, otherwise the configuration
+     is not valid. In &hawk;, primitives cannot be created or modified
+     while creating a group. Before adding a group, create primitives, 
+     and configure them as desired. For details, refer to
+     <xref linkend="sec.conf.hawk2.rsc.primitive"/>.
+    </para>
+   </note>
+   <!--taroth 2015-11-30: **missing functionality: once created, you can't rename a group or 
+    change its children **-->
+   <procedure xml:id="pro.conf.hawk2.group">
+    <title>Adding a Resource Group</title>
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para>From the left navigation bar, select <menuchoice>
+      <guimenu>Resources</guimenu>
+      <guimenu>Add a resource</guimenu>
+      <guimenu>Group</guimenu>
+     </menuchoice>.</para>
+    </step>
+    <step>
+     <para>
+      Enter a unique <guimenu>Group ID</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      To define the group members, select one or multiple entries in the list
+      of <guimenu>Children</guimenu>. <remark>taroth 2015-11-30: krig: how does resorting group
+       members work in hawk2? I did not find a way to change the sorting order...<!-- To define the order
+      of the group members, you currently need to add and remove them in the
+      order you desire.--></remark>
```

Comment:
So, caveat: This only works with the maintenance update as the UI was buggy and always re-sorted the resources in the GM version: Resort group members by dragging and dropping them into the order you want. Drag and drop using the "handle" on the right.